fdbserver/LatencyBandConfig.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * LatencyBandConfig.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2019 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBSERVER_LATENCYBANDCONFIG_H
+#define FDBSERVER_LATENCYBANDCONFIG_H
+#pragma once
+
+#include "fdbclient/FDBTypes.h"
+#include "fdbclient/JSONDoc.h"
+
+struct LatencyBandConfig {
+	struct RequestConfig {
+		std::set<double> bands;

```

Comment:
consider making this a vector

Reply:
I recall a reason why I didn't do this -- the equality operator compares the bands set to see if they are the same.